recipes/hexdump/all/conanfile.py

```diff
@@ -0,0 +1,29 @@
+import os
+import glob
+
+from conans import ConanFile, tools
+
+class HexdumpConan(ConanFile):
+    name = "hexdump"
```

Comment:
```suggestion
    name = "fernandovelcic-hexdump"
```

Because it's not the official hexdump project: https://github.com/conan-io/conan-center-index/blob/master/docs/faqs.md#what-is-the-policy-on-recipe-name-collisions
Same for folder name.

Reply:
Hello, thanks for the review, which is the official hexdump project in this case? I forked the project because the original seems to be abandoned.

Reply:
The official hexdump is listed on https://linux.die.net/man/1/hexdump 

> The hexdump command is part of the util-linux-ng package and is available from ftp://ftp.kernel.org/pub/linux/utils/util-linux-ng/. 

Your project is a fork from another project, not a tool, but a library, but still there is a name collision.

The original header-only is zmb3/hexdump. There is no problem packaging the original, we can add patches if there are bugs. For new forks, they should be considered as "new official" by the community (people which are related to the original project).

In this case, I would consider packaging zmb3/hexdump, and adding patches if necessary. The package name must be zmb3-hexdump, because again, the original hexdump is a Linux tool, not a library.

Reply:
thanks a lot, I'm pushing all the changes